tests/v1.0/pass/wildcard-overlay-example.yaml

```diff
@@ -0,0 +1,12 @@
+overlay: 1.0.0
+info:
+  title: Update many objects at once
+  version: 1.0.0
+actions:
+  - target: $.paths.*.get
+    update:
+      x-safe: true
+  - target: $.paths.*.get.parameters[?@.name=='filter' && @.in=='query']
+    update:
+      schema:
+        $ref: '/components/schemas/filterSchema'
```

Comment:
```suggestion
        $ref: '#/components/schemas/filterSchema'
```

Reply:
Oh no, that typo comes from the spec, `1.0.0.md` 😱

Reply:
* #87